sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -85,203 +85,267 @@ client = FormRecognizerClient(endpoint, credential)
+
+#### RecognizedForm
+`RecognizedForm` is what our service returns when it recognizes a [form](#recognizing-custom-forms "Recognizing Custom Forms"). It contains the type of the form(`form_type`),
+the [fields](#FormField "FormField") found in the form (`fields`), and information about each [page](#FormPage "FormPage") in the form document (`pages`).
+
+#### FormField
+Each field in a form contains the unique name of the field (`name`), information about the labels and values (`label_data` and `value_data`), and its own strongly-typed recognized value (`value`).
+
+#### FormPage
+Contains the recognized metadata, text lines (`lines`), and tables (`tables`) found on a single page of the form document.
 
 ### Long-Running Operations
 Long-running operations are operations which consist of an initial request sent to the service to start an operation,
-followed by polling the service at intervals to determine whether the operation has completed or failed, and if it has 
+followed by polling the service at intervals to determine whether the operation has completed or failed, and if it has
 succeeded, to get the result.
 
-Methods that train models or extract values from forms are modeled as long-running operations.  The client exposes 
-a `begin_<method-name>` method that returns an `LROPoller`.  Callers should wait for the operation to complete by 
-calling `result()` on the operation returned from the `begin_<method-name>` method.  Sample code snippets are provided 
+Methods that train models or recognize values from forms are modeled as long-running operations. The client exposes
+a `begin_<method-name>` method that returns an `LROPoller`. Callers should wait for the operation to complete by
+calling `result()` on the operation returned from the `begin_<method-name>` method. Sample code snippets are provided
 to illustrate using long-running operations [below](#Examples).
 
-### Training models
-Using the `CustomFormClient`, you can train a machine-learned model on your own form type.  The resulting model will 
-be able to extract values from the types of forms it was trained on.
+### FormTrainingClient
+A `FormTrainingClient` is the Form Recognizer interface to use for creating and managing custom machine-learned models.
+It provides operations for training models on forms you provide and operations for viewing and deleting models, as well as
+understanding how close you are to reaching subscription limits for the number of models you can train.
+
+#### Training models
+Using the `FormTrainingClient`, you can train a machine-learned model on your own form types. The resulting model will
+be able to recognize values from the types of forms it was trained on. If you are not looking to train your own models, you can
+use the service's [labelling tool][fr-labelling-tool]. There are multiple form types available to train your custom model with,
+but be sure to deploy your models only on forms that are of the same type as the model's training forms.
```

Comment:
what about just "use" instead of "deploy"?

Reply:
I'm not sure I understand the intent of this sentence ...

Reply:
I wanted to make clear that you should only recognize forms with type that the custom model can recognize. I'll try reworking this so it makes more sense